verticapy/learn/memmodel.py

```diff
@@ -0,0 +1,1036 @@
+    Predicts using a linear regression model and the input attributes.
+
+    Parameters
+    ----------
+    X: list / numpy.array
+        Data on which to make the prediction.
+    coefficients: list / numpy.array
+        List of the model's coefficients.
+    intercept: float
+        The intercept or constant value.
+    method: str, optional
+        The model category, one of the following: 'LinearRegression', 'LinearSVR', 
+        'LogisticRegression', or 'LinearSVC'.
+    return_proba: bool, optional
+        If set to True and the method is set to 'LogisticRegression' or 'LinearSVC', 
+        the probability is returned.
+
+    Returns
+    -------
+    numpy.array
+        Predicted values
+    """
+    check_types([("X", X, [list,],), 
+                 ("coefficients", coefficients, [list,],),
+                 ("intercept", intercept, [float, int,],),
+                 ("method", method, ["LinearRegression", "LinearSVR", "LogisticRegression", "LinearSVC"],),
+                 ("return_proba", return_proba, [bool],),])
+    result = intercept + np.sum(np.array(coefficients) * np.array(X), axis=1)
+    if method in ("LogisticRegression",):
+        result = 1 / (1 + np.exp(- (result)))
+    elif method in ("LinearSVC",):
+        result =  1 - 1 / (1 + np.exp(result))
+    else:
+        return result
+    if return_proba:
+        return np.column_stack((1 - result, result))
+    else:
+        return np.where(result > 0.5, 1, 0)
+
+# ---#
+def sql_from_coef(X: list, 
+                  coefficients: list, 
+                  intercept: float, 
+                  method: str = "LinearRegression",):
+    """
+    ---------------------------------------------------------------------------
+    Returns the SQL code needed to deploy a linear model using its attributes.
+
+    Parameters
+    ----------
+    X: list
+        The name or values of the input predictors.
+    coefficients: list
+        List of the model's coefficients.
+    intercept: float
+        The intercept or constant value.
+    method: str, optional
+        The model category, one of the following: 'LinearRegression', 'LinearSVR', 
+        'LogisticRegression', or 'LinearSVC'.
+
+    Returns
+    -------
+    str
+        SQL code
+    """
+    check_types([("X", X, [list],), 
+                 ("coefficients", coefficients, [list],),
+                 ("intercept", intercept, [float, int,],),
+                 ("method", method, ["LinearRegression", "LinearSVR", "LogisticRegression", "LinearSVC"],),])
+    assert len(X) == len(coefficients), ParameterError("The length of parameter 'X' must be equal to the number of coefficients.")
+    sql = [str(intercept)] + [f"{coefficients[idx]} * {(X[idx])}" for idx in range(len(coefficients))]
+    sql = " + ".join(sql)
+    if method in ("LogisticRegression",):
+        return f"1 / (1 + EXP(- ({sql})))"
+    elif method in ("LinearSVC",):
+        return f"1 - 1 / (1 + EXP({sql}))"
```

Comment:
The same here, you can consolidate lines 138 to 141.

Reply:
Done;